stage.py

```diff
@@ -196,16 +153,19 @@ def get_ideal_times_from_metrics(self):
           total_disk_throughput_Bps += disk_metrics.effective_throughput_Bps()
 
     num_executors = len(executor_id_to_metrics)
-    num_cores_per_executor = 8
-    ideal_cpu_s = float(total_cpu_millis) / (num_executors * num_cores_per_executor * 1000)
 
-    # Compute how many bytes the job thinks it sent over the network. If that's
-    # 0, all of the network traffic was control messages (and not related to the job's
-    # completion time) so the ideal network time is 0.
-    if self.get_network_mb() > 0:
-      ideal_network_s = float(total_network_bytes_transmitted) / total_network_throughput_Bps
-    else:
-      ideal_network_s = 0
+    ideal_cpu_s = self.__get_ideal_cpu_s(
+      total_cpu_millis_os_counters = total_cpu_millis,
```

Comment:
They're all integers -- so passing them as keyword arguments helps avoid typos where things are passed in the wrong order.